packages/gasket-plugin-docusaurus/lib/configure.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const timing = { before: ['@gasket/plugin-docs'] };
+
+async function handler(gasket, config) {
+  const { docusaurus = {} } = config;
+  const { docsDir = '.docs' } = docusaurus;
+
+  // Docusaurus requires a ./docs folder
+  // config.docs.outputDir is required to be docs
```

Comment:
It would probably be good to add a warning if the user as a custom `docs.outputDir` configured in the gasket.config.js. Something like:

```js
if(config.docs && config.docs.outputDir) {
  gasket.logger.warning('Custom config for `docs.outputDir` found. Instead use `docusaurus.docsDir`.')
}
```

Reply:
Yes I remember you talking about that. I'll add that in first thing tomorrow!